Tools/scripts/build_all.sh

```diff
@@ -15,7 +15,7 @@ BOARDS="sitl linux Pixhawk1"
 
 for b in $BOARDS; do
     echo "Testing $b build"
-    ./waf configure --board $b
+    ./waf configure --board "$b"
```

Comment:
If we *ever* put a space in a board name then *VAST* numbers of things are going to break :-)